### PR TITLE
Use github url for RHEL6 testing

### PR DESCRIPTION
**Pull request by lakshmi-kannan**

### CODE REVIEW

**Comment by manasdk** [comment]
Can you also modify the action that generate equivalent rules for release branches

**Comment by manasdk** [comment]
+1

bad things are often needed :P

**Comment by lakshmi-kannan** [comment]
> Can you also modify the action that generate equivalent rules for release branches

will do.

**Comment by Kami** [comment]
LGTM

File: rules/st2workroom_test_master_el7.yaml [hunk 1]
```diff
@@ -34,3 +34,5 @@
             instance_type: "m3.large"
             distro: "RHEL7"
             pkg_st2: true
+            install_url: "https://raw.githubusercontent.com/StackStorm/st2workroom/master/script/bootstrap-st2"
```
Comment (Kami) [inline rules/st2workroom_test_master_el7.yaml:37]: inb4, github changes supported protocols and ciphers and this stuff breaks again, right? :P

Comment (lakshmi-kannan) [inline rules/st2workroom_test_master_el7.yaml:37]: One problem at a time :P

**Pull request merged**